datamodel/extensions/extension_manager.py

```diff
@@ -80,7 +80,7 @@ def load_extension(
         files.sort()
         for file in files:
             filename = os.fsdecode(file)
-            if filename.endswith(".sql"):
+            if filename.endswith(".sql") or filename.endswith(".SQL") :
```

Comment:
PP: Für SQL und alle anderen typen welche noch geprüft werden.
```suggestion
            if filename.lower().endswith(".sql"):
```